clients/privacy-center/components/consent/ConsentDescription.tsx

```diff
@@ -0,0 +1,42 @@
+import { Box, Text, TextProps } from "@fidesui/react";
```

Comment:
Also a refactor, pulling out the description part of `consent.tsx` and then letting this component figure out if we are notice or config driven